applications/validate_camera_fov.py

```diff
@@ -96,12 +114,24 @@ def main():
     print("Avg. edge radius = {0:.3f} cm\n".format(rEdgeAvg))
 
     # Now plot the camera as well
-    fig = camera.plotPixelLayout()
-    plotFileName = label + "_" + telModel.name + "_pixelLayout"
-    plotFile = outputDir.joinpath(plotFileName)
-    for f in ["pdf", "png"]:
-        fig.savefig(str(plotFile) + "." + f, format=f, bbox_inches="tight")
-    print("\nPlotted camera in {}\n".format(plotFile))
+    try:
+        pixelIDsToPrint = int(args.printPixelsID)
+        if pixelIDsToPrint == 0:
```

Comment:
Why don't we require as an argument on the command line to run `--printPixelsID -1` instead of `--printPixelsID 0`? This would not require this line.

Reply:
I thought about it, but that's less intuitive to the user and I preferred to hide the ugliness of the implementation from the user. Also, in the case, if the user sets it to zero, it would print just the zero pixel which would be strange.